decidim-blogs/spec/forms/decidim/blogs/admin/post_form_spec.rb

```diff
@@ -52,15 +54,35 @@ module Admin
             subject.map_model(post)
           end
 
-          it "does not assign the user group for normal users" do
-            expect(subject.user_group_id).to be_nil
+          it "assigns current_user as user_group_id" do
+            expect(subject.user_group_id).to eq("current_user")
+          end
+
+          it "assigns user name as author" do
+            expect(subject.author).to be_nil
+          end
+
+          context "when author is an organization" do
+            let(:author) { current_organization }
+
+            it "assigns 'current_organization' as user_group_id" do
+              expect(subject.user_group_id).to eq("current_organization")
+            end
+
+            it "assigns current_organization as author" do
+              expect(subject.author).to eq(author)
+            end
           end
 
           context "when the author is a group" do
             let(:author) { create(:user_group, :verified, organization: current_organization) }
```

Comment:
```suggestion
            let(:author_id) { user_group.id }
```
`user_group` is already defined on top of this file. Here we need to ensure that we are sending as a param the `id` of the user group

Reply:
Done